contracts/MigrationZapper.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+import {IStaking} from "./interfaces/IStaking.sol";
+import {IMigrator} from "./interfaces/IMigrator.sol";
+import {IMintableERC20} from "./interfaces/IMintableERC20.sol";
+
+contract MigrationZapper {
+    IMintableERC20 public immutable ogv;
+    IMintableERC20 public immutable ogn;
+
+    IMigrator public immutable migrator;
+    IStaking public immutable ognStaking;
+
+    address public immutable governor;
+
+    error NotGovernor();
+
+    constructor(address _ogv, address _ogn, address _migrator, address _ognStaking, address _governor) {
+        ogv = IMintableERC20(_ogv);
+        ogn = IMintableERC20(_ogn);
+        migrator = IMigrator(_migrator);
+        ognStaking = IStaking(_ognStaking);
+
+        governor = _governor;
+    }
+
+    function initialize() external {
```

Comment:
Fortuity, `initialize` can't do any damage, so it's okay if it's called multiple times, but this should have an initializer modifier.

Reply:
Alternatively, if no proxy, this initialize code could move into constructor.